pandas_ml/confusion_matrix/abstract.py

```diff
@@ -0,0 +1,515 @@
+#!/usr/bin/python
+# -*- coding: utf8 -*-
+
+import numpy as np
+import pandas as pd
+# import matplotlib as mpl
+import matplotlib.pylab as plt
+import collections
+
+from pandas_ml.confusion_matrix.stats import binom_interval, class_agreement, prop_test
+
+from pandas_ml.confusion_matrix.utils import (
+    DISPLAY_SUM_DEFAULT, BACKEND_DEFAULT,
+    TRUE_NAME_DEFAULT, PRED_NAME_DEFAULT,
+    CLASSES_NAME_DEFAULT,
+    SUM_NAME_DEFAULT, COLORBAR_TRIG,
+    Backend)
+
+
+class ConfusionMatrixAbstract(object):
+    """
+    Abstract class for confusion matrix
+
+    You shouldn't instantiate this class.
+    You might instantiate ConfusionMatrix or BinaryConfusionMatrix classes
```

Comment:
Isn't it possible to distinguish whether ``ConfusionMatrix`` or ``BinaryConfusionMatrix`` is preferable by count the number of unique values?